lib/butler/adapters/console.ex

```diff
@@ -37,10 +36,14 @@ defmodule Butler.Adapters.Console do
   def format_response(msg) when is_binary(msg) do
     format_response({:text, msg})
   end
-  def format_response({:code, msg}),  do: {:ok, "\n#{msg}"}
-  def format_response({:text, msg}),  do: {:ok, "#{msg}"}
-  def format_response({:quote, msg}), do: {:ok, ">#{msg}"}
-  def format_response(response), do: {:error, response}
+  def format_response({:code, msg}),  do: "\n#{msg}"
+  def format_response({:text, msg}),  do: "#{msg}"
+  def format_response({:quote, msg}), do: ">#{msg}"
+  def format_response(_), do: ""
+
+  defp mention_user(user) do
+    "@#{user}: "
```

Comment:
We're just simplifying the api so we don't have to lift the value out of the tuple.